### PR TITLE
Replace fpm cookery with rpm spec files

### PR DESCRIPTION
**Pull request by Metallion**
We plan to move away from fpm cookery and back to standard rpm spec files. Rpm spec files are much better documented, more well known in the industry and will allow us more control over the packages.

This PR replaces the OpenVNet packages. The third party packages will be done at a later time.

### CODE REVIEW

**Comment by Metallion** [comment]
The integration test failed once because I was running it manually with the wrong environment variables. Starting the CI again from the beginning and letting each job run their downstream automatically fixed it.

File: deployment/packagebuild/packages.d/vnet/openvnet.spec [hunk 1]
```diff
@@ -0,0 +1,217 @@
+# This is a little trick to allow the rpmbuild command to define a suffix for
+# development (non stable) versions.
+%define release 2
+%{?dev_release_suffix:%define release %{dev_release_suffix}}
+
+Name: openvnet
+Version: 0.7%{?dev_release_suffix:dev}
+Release: %{release}.el6
```
Comment (hansode) [inline deployment/packagebuild/packages.d/vnet/openvnet.spec]: We can use `%{dist}` macro for rpmspec.

```
Release: %{release}%{?dist}
```

**Comment by hansode** [comment]
:+1:

**Comment by Metallion** [comment]
The integration test failure at Aug 22, 2015 3:28:45 AM JST was a router p2v race condition that I've seen happening on develop in nightly builds.

The one at Aug 22, 2015 3:47:13 AM JST was a vnet edge failure that I haven't been able to reproduce.

While these failures do worry me, I will merge since I haven't been able to reproduce them and we had 4 successful integration tests since then. I highly suspect the failures to not be related to the rpm packages in this PR.

**Pull request merged**